main/src/ca/uwaterloo/flix/language/phase/constraintgeneration/TypeContext.scala

```diff
@@ -31,11 +31,10 @@ import scala.collection.mutable
 class TypeContext {
 
   private object ScopeConstraints {
-
     /**
-      * Creates an empty ScopeConstraints with no associated region.
+      * The empty ScopeConstraints with no associated region.
       */
-    def empty: ScopeConstraints = new ScopeConstraints(None)
+    val empty: ScopeConstraints = new ScopeConstraints(None)
```

Comment:
ScopeConstraints has a mutable field, so this will probably break something.

Reply:
Oh okay